sdk/examples/c2pa.toml

```diff
@@ -0,0 +1,172 @@
+# c2pa-rs Configuration File
+
+# Version information.
+version_major = 1
+version_minor = 0
+
+# # Trust settings for certificate validation.
+# [trust]
+# # String to user-provided trust anchors (PEM format).
+# user_anchors = ""
+# # String to system trust anchors (PEM format).
+# trust_anchors = ""
```

Comment:
Like Gavin hinted at.  May this could also take a URL and the validation code could fetch if it found a URL here instead of an anchor list.  Not a requirement for this PR.